services/web-app/components/color-palette/color-palette.module.scss

```
@@ -0,0 +1,371 @@
+  background-color: theme.$border-inverse;
+  outline: 1px solid theme.$border-inverse;
+}
+
+.group-container .color-palette-color {
+  border-bottom: 1px solid theme.$border-subtle-01;
+
+  &:last-of-type {
+    border-bottom: none;
+  }
+}
+
+// responsive group option dividers
+.group-container + .group-container .group-option::before {
+  position: absolute;
+  top: 0;
+  left: -1px;
+  width: 1px;
+  height: 100%;
+  background-color: theme.$border-subtle-01;
+  content: '';
+
+  @include breakpoint.breakpoint-down('md') {
+    //all indiv rows, so don't need a left "border"
+    display: none;
+  }
+}
+
+.dark-mode .group-container .color-palette-color {
+  border-bottom: 1px solid theme.$border-inverse;
+  outline: none;
+}
+
+.group-option {
+  position: relative;
+  height: 3rem;
+  align-content: center;
+  padding: spacing.$spacing-06 0 spacing.$spacing-03 spacing.$spacing-05;
+  background-color: var(--cds-focus-inverse, theme.$layer-01);
+  font-size: 12px;
+  font-weight: 600;
+}
+
+.dark-mode .group-option {
+  background: theme.$border-inverse;
+  color: var(--cds-focus-inverse, theme.$layer-01);
+}
+
+.dark-mode .group-container + .group-container .group-option::before {
+  background-color: theme.$icon-secondary;
+}
+
+// GROUP
+.group-container + .group-container:nth-child(4) .group-option::before {
+  @include breakpoint.breakpoint-up('lg') {
+    //4 starts a new row, so it doesn't need a left "border"
+    display: none;
+  }
+}
+
+.group-container + .group-container:nth-child(3) .group-option::before,
+.group-container + .group-container:nth-child(5) .group-option::before {
+  @include breakpoint.breakpoint-between('md', 'lg') {
+    // 3/5 start a new row, so it don't need a left "border"
+    display: none;
+  }
+}
+
+// SEQUENTIAL
+.sequential-controls {
+  border-bottom: 1px solid theme.$border-subtle-01;
+  margin-bottom: 0;
+}
+
+.sequential-container {
+  display: grid;
+  width: 66.66%;
+  grid-template-columns: 1fr 1fr;
+
+  @include breakpoint.breakpoint-down('lg') {
+    width: 100%;
+  }
+
+  @include breakpoint.breakpoint-down('md') {
+    grid-template-columns: 1fr;
+  }
+}
+
+.sequential {
+  column-gap: 0;
+  grid-template-columns: 1fr;
+  row-gap: 0;
+}
+
+// column divider
+.sequential:nth-child(even) {
+  @include breakpoint.breakpoint('md') {
+    border-left: 1px solid theme.$layer-02;
+  }
+}
+
+// group option divider
+.sequential-container .palettes-container:nth-child(even) .group-option::before {
+  position: absolute;
+  top: 0;
+  left: -1px;
+  width: 1px;
+  height: 100%;
+  background-color: theme.$border-subtle-01;
+  content: '';
+
+  @include breakpoint.breakpoint-down('md') {
+    //all indiv rows, so don't need a left "border"
+    display: none;
+  }
+}
+
+.gradient-purple {
+  background: linear-gradient(
+    to bottom,
+    #f6f2ff,
+    #e8daff,
+    #d4bbff,
+    #be95ff,
+    #a56eff,
+    #8a3ffc,
+    #6929c4,
+    #491d8b,
+    #31135e,
+    #1c0f30
```

```suggestion
    #f6f2ff,
    #e8daff,
    #d4bbff,
    #be95ff,
    #a56eff,
    #8a3ffc,
    #6929c4,
    #491d8b,
    #31135e,
    #1c0f30
```
```suggestion
    colors.$purple-10,
    colors.$purple-20,
    colors.$purple-30,
    colors.$purple-40,
    colors.$purple-50,
    colors.$purple-60,
    colors.$purple-70,
    colors.$purple-80,
    colors.$purple-90,
    colors.$purple-100
```

Might as well swap these for the variables while we're at it. Same for the other gradients below.